tests/integration/test_permissions.py

```diff
@@ -58,6 +58,7 @@ def _create_and_index_record(record):
 def sample_record(app):
     record = {
         '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'authors': [],
```

Comment:
> This is exactly what I just wrote.

Not explicitly. So at least it was clear to @zzacharo .

> I can apply a dummy one.

Yes, please do. :+1: